packages/next/server/config-shared.ts

```diff
@@ -140,6 +140,7 @@ export type NextConfig = { [key: string]: any } & {
     isrMemoryCacheSize?: number
     nftTracing?: boolean
     concurrentFeatures?: boolean
```

Comment:
It looks like this shouldn't be in the experimental section, related code https://github.com/vercel/next.js/blob/93bac0f2b710a0a5ea3d2aafb9d53e80d68df37d/packages/next/build/webpack-config.ts#L663

Reply:
Good catch. Thank you.